### ISSUE TITLE
Auto release is failing on repository without tags, and once tags are added, it will fail that tags already exist.

### ISSUE THREAD

**Issue by vincentbriglia** [intuit/auto#1401]
**Describe the bug**

essentially this one, but i feel it's a regression: https://github.com/intuit/auto/issues/886

**To Reproduce**

find https://github.com/LEGO/poc-yarn-lerna-workspaces-auto-typescript @hipstersmoothie I invited you as an external collaborator. and check the next branch

publishing from CI fails with 

```
Error: Running command 'git' with args [describe, --tags, --abbrev=0, ] failed
fatal: No names found, cannot describe anything.
    at ChildProcess.child.on.code (/snapshot/auto/node_modules/@auto-it/core/dist/utils/exec-promise.js:49:24)
    at ChildProcess.emit (events.js:198:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
   script returned exit code 1
```

publishing from local will succeed and a version will be tagged

subsequent releases on ci will fail, because it will complain that a tag already exists

**Expected behavior**

expect to make versions off of conventional commits
expected first CI run to publish version @ packages/*/package.json or 0.0.0
independent monorepo packages to be bumped on each commit or pr into next

**Comment by hipstersmoothie** [comment]
I think you're right and this is a regression. I'm going to make a PR with a canary version though should fix the issue. 

A few things I noticed from your rc:

1. the release plugins are meant to be used 1 at a time in most cases. So you only need the `npm` plugin and not the `git-tag` plugin
2. Order does matter in advanced cases (custom plugins) but for all of the official ones you are pretty safe with any ordering

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v9.48.2`](https://github.com/intuit/auto/releases/tag/v9.48.2) :rocket:
